### DISCUSSION TITLE
How to Hide Already Selected Tags from Antd Select

### DISCUSSION THREAD

**Discussion by Rajgupta7080** [ant-design/ant-design#46422] [Q&A]
`
import React, { useState } from 'react';
import { Select } from 'antd';
const OPTIONS = ['Apples', 'Nails', 'Bananas', 'Helicopters'];
const App = () => {
  const [selectedItems, setSelectedItems] = useState([]);
  const filteredOptions = OPTIONS.filter((o) => !selectedItems.includes(o));
  return (
    <Select
      mode="tags"
      placeholder="Inserted are removed"
      value={selectedItems}
      onChange={setSelectedItems}
      style={{
        width: '100%',
      }}
      options={filteredOptions.map((item) => ({
        value: item,
        label: item,
      }))}
    />
  );
};
export default App;
`

**Comment by Wxh16144** [comment]
Maybe you can do it with `tagRender`? https://ant.design/components/select#select-demo-custom-tag-render

**Reply by Rajgupta7080** [reply]
@Wxh16144 The problem is not about tags it is about dropdown why it is not removing from options?
![image](https://github.com/ant-design/ant-design/assets/107268454/c9e315de-efc1-46f9-b5a4-c7e77236feb2)

**Reply by Wxh16144** [reply]
Implemented in a controlled manner to get filtered options

**Reply by Rajgupta7080** [reply]
@Wxh16144 So how can I hide selected options from the dropdown any other approach or code you can share it would be great help.

**Reply by AzharUddinSheikh** [reply]
@Rajgupta7080 you need to define a variable or state `already selected` as given in the example [here](https://ant.design/components/select#select-demo-hide-selected) it's work for me
